test/helper/helper.go

```diff
@@ -164,3 +170,52 @@ func TestServer() *server.Yorkie {
 	}
 	return y
 }
+
+func getDocumentKey(name string) string {
```

Comment:
https://go.dev/doc/effective_go#Getters

```suggestion
func testDocumentKey(name string) string {
```